lisa/tools/nvmecli.py

```diff
@@ -30,6 +34,18 @@ def _install_from_src(self) -> None:
         code_path = tool_path.joinpath("nvme-cli")
         make.make_and_install(cwd=code_path)
 
+    def create_namespace(self, namespace: str) -> ExecutableResult:
+        cmd_result = self.run(f"create-ns {namespace}", shell=True, sudo=True)
+        return cmd_result
+
+    def delete_namespace(self, namespace: str, id: int) -> ExecutableResult:
+        cmd_result = self.run(f"delete-ns -n {id} {namespace}", shell=True, sudo=True)
+        return cmd_result
+
+    def detach_namespace(self, namespace: str, id: int) -> ExecutableResult:
+        cmd_result = self.run(f"detach-ns -n {id} {namespace}", shell=True, sudo=True)
```

Comment:
It can be like below. BTW, why not parse the result, and return success or fail?
`return self.run(f"detach-ns -n {id} {namespace}", shell=True, sudo=True)`